src/utils/utils.cpp

```diff
@@ -838,3 +838,18 @@ QString trimNULL(QString& s) {
     }
     return s;
 }
+
+bool openUrl(QUrl url) {
+#ifdef Q_OS_LINUX
+    // On linux openUrl will open the default browser via xgd-open, at this time LD_LIBRARY_PATH may affect xdg-open.
```

Comment:
On linux openUrl will open the default browser via xgd-open. If the client is packaged as AppImage format,  LD_LIBRARY_PATH is set to package location and may affect xdg-open.